apps/projects/app/store/app.js

```diff
@@ -1,2 +0,0 @@
-import Aragon from '@aragon/api'
-export const app = new Aragon()
```

Comment:
I don't think this should actually be removed, it's used in at least one place.

Reply:
omg, yes it is, sorry about that. Re-added.